ocean_data_parser/read/vocabularies/seabird_variable_attributes.json

```diff
@@ -1338,7 +1343,8 @@
     },
     "latitude": {
         "long_name": "Latitude",
-        "units": "deg",
+        "bodc_alternative_label": "Lat",
+        "units": "degree north",
```

Comment:
I would standardize those to the recommanded CF units `degrees_north`

Reply:
Agreed !

Reply:
Sounds good I made the changes